src/pages/DatasetRegistration.js

```diff
@@ -551,89 +317,118 @@ class DatasetRegistration extends Component {
     this.props.history.goBack();
   };
 
+  createProperties = () => {
```

Comment:
this can create a partial set of properties if it doesn't receive them all (just like how the backend only actually requires datasetName). However, we have other code that blocks form submission in case the required fields on the form aren't filled out